Components/ScrewMotor.py

```
@@ -0,0 +1,73 @@
+import RPi.GPIO as GPIO          # Import GPIO library for Raspberry Pi
+from time import sleep, time     # Import sleep for delays and time for timestamps
```

Ruff (0.11.9)

`time.sleep` imported but unused

Remove unused import

(F401)